app/scripts/filters/canI.js

```diff
@@ -29,6 +29,9 @@ angular
       'imageStreams': [
         {group: '', resource: 'imagestreams', verbs: ['update', 'delete']}
       ],
+      'instances': [
+        {group: 'servicecatalog.k8s.io', resource: 'instances', verbs: ['update', 'delete']}
```

Comment:
This will need to change to `'serviceinstances'` when https://github.com/openshift/origin/pull/16150 is merged.

Reply:
going to update line 32 to match with ```serviceinstances```,  currently still retains the "instances" string while I walk/run through the changes again